roles/linux_ansible_init/files/ansible-init.py

```diff
@@ -103,3 +124,6 @@ def ansible_exec(cmd, *args, **kwargs):
             "127.0.0.1,",
             playbook["name"]
         )
+
+
+create_sentinel_file()
```

Comment:
I don't think this needs to be a function - the rest of the script is not except for pieces that are actually reused.